config/owasp/suppressions.xml

```diff
@@ -6,7 +6,6 @@
       and hence it is suppressed (https://github.com/jeremylong/DependencyCheck/issues/2952)
```

Comment:
can you try to remove CVE-2018-1258 and run the build? as it was old false positive

Reply:
Tried removing the supression @amitchidrewar1301 but it raises  CVE-2018-1258 in security checks. So adding the supression again